SConstruct

```diff
@@ -412,6 +412,14 @@ if env["prereqs"]:
         if env["history"]:
             client_env.Append(CPPDEFINES = ["HAVE_HISTORY"])
 
+        if env["PLATFORM"] == 'darwin':
+            client_env.Append(FRAMEWORKS = "OpenGL")
+            client_env.AppendUnique(LIBS = ["GLEW"])
+        elif env["PLATFORM"] == 'posix':
+            opengl_config = check_output(["pkg-config", "--libs", "--cflags", "gl", "glew"])
+            opengl_flags = client_env.ParseFlags(opengl_config)
+            client_env.MergeFlags(opengl_flags)
```

Comment:
Please use `client_env.ParseConfig("pkg-config --libs --cflags gl glew")` instead. (Documented in the [SCons manual](http://scons.org/doc/production/HTML/scons-man.html).)